flusher.go

```diff
@@ -147,6 +147,11 @@ func (s *Server) flushSink(
 				maxNameLengthCount += 1
 				continue metricLoop
 			}
+
+			for k, v := range sink.extraTags {
+				metric.Tags = append(metric.Tags, fmt.Sprintf("%s:%s", k, v))
+			}
+
```

Comment:
I think we should add tags _after_ striping tags. It doesn't make much sense to add a tag, and then strip it. Stripping then adding would allow replacing a tag.

Also could you add a unit test verifying this? Thanks!

Reply:
Done